scripts/check-url-locale.js

```diff
@@ -12,14 +12,11 @@
 import fs from "node:fs/promises";
 import * as path from "node:path";
 import { fdir } from "fdir";
-import ora from "ora";
 import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
 import { fromMarkdown } from "mdast-util-from-markdown";
 import { visit } from "unist-util-visit";
 
-const spinner = ora().start();
```

Comment:
Instead of removing `ora`, let's set `no_tty` in the lefthook config: https://lefthook.dev/configuration/no_tty.html

Reply:
Well spotted, I didn't know about this config

Reply:
See https://github.com/mdn/translated-content/pull/26482/commits/31d2eb49013fff33114608e6bb32abbdd16de0e9